File: C/Vetores/Maior_posicao.c
#include <stdio.h>
#include <string.h>


int main ()
{

    int N, i, pares, posicao;
    double maior;

    printf ("Quantos numeros voce vai digitar? ");
    scanf ("%d", &N);

    double vet[N];

    for (i = 0; i < N; i++)
    {
        printf ("Digite um numero: ");
        scanf ("%lf", &vet[i]);
    }

    maior = 0;

    for (i = 0; i < N; i++){
        if (vet[i] > maior){
            maior = vet[i];
            posicao = i;
        }
    }
    printf ("\nMAIOR VALOR = %.1lf\n", maior);
    printf ("POSICAO DO MAIOR VALOR = %d\n", posicao);


    return 0;
}

File: Python/Vetores/Maior_posicao.py
N = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(N)]

for i in range(0, N):
    vet[i] = float(input("Digite um numero: "))


maior = 0
posicao = 0
for i in range(0, N):
    if vet[i] > maior:
        maior = vet[i]
        posicao = i

print()
print(f"MAIOR VALOR = {maior:.1f}")
print(f"POSICAO DO MAIOR VALOR = {posicao}")